### PR TITLE
Send Email to Admin on new User Registration

### PR DESCRIPTION
**Pull request by thomascoe**
This fixes issue #21. Please squash merge when approved

### CODE REVIEW

File: front-end-pages/registration-form/submit.php [hunk 1]
```diff
@@ -32,22 +31,29 @@
...
+    //Create Customer in G Suite
+    $newUser = userFactory($username, $email, $firstname, $lastname, $password, $customer->id, $plan->name, $status, $recurring=='checked', $expiration);
+    createUser($newUser);
 
+    // Have Slack invite the user
     sendSlackInvite($email, $firstname, $lastname);
 
+    // Send email to admin about the new user
+    // TODO: Make this admin email configurable somewhere?
+    $adminemail = 'thomas@decaturmakers.org';
```
Comment (cmowenby) [inline front-end-pages/registration-form/submit.php:49]: This can be done when I redo the integrations page to word with WP settings.

**Review by cmowenby — APPROVED** [review]
This also looks great! Good work!

**Comment by ryanstrat** [comment]
I'm not a huge fan of the admin email being hardcoded. We should pull a list of admins from G-Suite or Wordpress and email the whole group.

The code here I think meets the requirements for MVP, so I support merging it, and opening an issue for enhancement.

**Comment by thomascoe** [comment]
I agree, the hardcoded email was not intended to be permanent. Ideally Craig can add this configuration when he does the WP settings. I'm not sure if there's anywhere in G Suite we can pull a list of admins from. I'll open another issue for that though and merge this PR for now.

**Pull request merged**